pandas/io/formats/format.py

```diff
@@ -1855,29 +1856,51 @@ def just(x):
     return result
 
 
-def _trim_zeros_complex(
-    str_complexes: np.ndarray, decimal: str = ".", na_rep: str = "NaN"
-) -> List[str]:
+def _trim_zeros_complex(str_complexes: np.ndarray, decimal: str = ".") -> List[str]:
     """
     Separates the real and imaginary parts from the complex number, and
     executes the _trim_zeros_float method on each of those.
     """
     return [
-        "".join(_trim_zeros_float(re.split(r"([j+-])", x), decimal, na_rep))
+        "".join(_trim_zeros_float(re.split(r"([j+-])", x), decimal))
         for x in str_complexes
     ]
 
 
+def _post_process_complex(complex_strings: List[str]) -> List[str]:
```

Comment:
why does this need to be a separate function? can't you do this in _trim_zeros_complex

Reply:
Yes that's better